### PR TITLE
login page created

### PR DESCRIPTION
**Pull request by provishalk**
login page

-- you mean signup??

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,24 +1,13 @@
-import logo from './logo.svg';
-import './App.css';
-
-function App() {
+import Registration from "./pages/Registration/Registration";
```
Comment (shivdotpy) [inline src/App.js]: Don't create a Pages folder, should be a container.

File: src/App.js [hunk 2]
```diff
@@ -1,24 +1,13 @@
...
+import 'bootstrap/dist/css/bootstrap.min.css';
+import 'alertifyjs/build/css/alertify.css';
+import alertify from 'alertifyjs';
+alertify.set('notifier','position', 'top-right');
```
Comment (shivdotpy) [inline src/App.js:5]: Do this in app.js so it will effect on whole application

File: src/App.js [hunk 3]
```diff
@@ -1,24 +1,13 @@
...
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <>
+      <Registration/>
```
Comment (shivdotpy) [inline src/App.js]: Where is routing?

File: src/pages/Registration/Registration.js [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+import alertify from 'alertifyjs';
+const Registration = () => {
+    const [name, setName] = useState("");
+    const [email, setEmail] = useState("");
+    const [password, setPassword] = useState("");
+    const onFormSubmitHandler = event => {
+        event.preventDefault();
+        const data = { name, email, password };
+        axios
+            .post("https://social-nodejs-be.herokuapp.com/api/user/signup", data)
```
Comment (shivdotpy) [inline src/pages/Registration/Registration.js]: Keep this Base URL in .env file
Base URL: https://social-nodejs-be.herokuapp.com

rest URL should be in a constant file:
/api/user/signup

File: src/pages/Registration/Registration.js [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+    return (
+        <div className="registration_container container">
+            <h1>SIGN UP</h1>
+            <div className="row">
+                <div className="col-md-6 m-auto">
+                    <Form onSubmit={onFormSubmitHandler} className="mt-4">
+                        <Form.Group className="mb-3" controlId="formBasicName">
+                            <Form.Control
+                                type="text"
+                                placeholder="Enter Full Name"
```
Comment (shivdotpy) [inline src/pages/Registration/Registration.js]: Put into constant file

File: src/pages/Registration/Registration.js [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+                                value={name}
+                                onChange={event => {
+                                    setName(event.target.value);
+                                }}
+                                required />
+                        </Form.Group>
+                        <Form.Group className="mb-3" controlId="formBasicEmail">
+                            <Form.Control
+                                type="email"
+                                placeholder="Enter email"
```
Comment (shivdotpy) [inline src/pages/Registration/Registration.js]: Put into constant file

File: src/pages/Registration/Registration.js [hunk 7]
```diff
@@ -0,0 +1,79 @@
...
+                                value={email}
+                                onChange={event => {
+                                    setEmail(event.target.value);
+                                }}
+                                required />
+                            <Form.Text className="text-muted">
+                                We'll never share your email with anyone else.
```
Comment (shivdotpy) [inline src/pages/Registration/Registration.js]: Put into constant file

File: src/pages/Registration/Registration.js [hunk 8]
```diff
@@ -0,0 +1,79 @@
...
+                            </Form.Text>
+                        </Form.Group>
+                        <Form.Group className="mb-3" controlId="formBasicPassword">
+                            <Form.Control
+                                type="password"
+                                placeholder="Password"
```
Comment (shivdotpy) [inline src/pages/Registration/Registration.js]: Put into constant file

File: src/pages/Registration/Registration.js [hunk 9]
```diff
@@ -0,0 +1,79 @@
...
+                                value={password}
+                                onChange={event => {
+                                    setPassword(event.target.value);
+                                }}
+                                required
+                            />
+                            <Form.Text className="text-muted">
+                                By clicking Sign Up, you agree to our Terms, Data Policy and Cookie Policy.
```
Comment (shivdotpy) [inline src/pages/Registration/Registration.js]: Put into constant file

File: src/pages/Registration/Registration.js [hunk 10]
```diff
@@ -0,0 +1,79 @@
...
+                            </Form.Text>
+                        </Form.Group>
+                        <Button variant="primary" type="submit" className="btn-success">
+                            Sign Up
```
Comment (shivdotpy) [inline src/pages/Registration/Registration.js]: Put into label file

File: src/pages/Registration/Registration.scss [hunk 11]
```diff
@@ -0,0 +1,9 @@
+.registration_container {
+    margin: auto;
+    margin-top: 2vh;
+
+    h1 {
+        display: flex;
```
Comment (shivdotpy) [inline src/pages/Registration/Registration.scss]: learn about mixins, so this can be reduced, you can ignore for now.

File: src/App.js (hunk 1 above)
Comment (provishalk) [reply src/App.js]: Done

File: src/App.js (hunk 3 above)
Comment (provishalk) [reply src/App.js]: Done

File: src/pages/Registration/Registration.js (hunk 4 above)
Comment (provishalk) [reply src/pages/Registration/Registration.js]: Done.

File: src/pages/Registration/Registration.scss (hunk 11 above)
Comment (provishalk) [reply src/pages/Registration/Registration.scss]: ok

File: src/pages/Registration/Registration.js (hunk 5 above)
Comment (provishalk) [reply src/pages/Registration/Registration.js]: done

File: src/pages/Registration/Registration.js (hunk 6 above)
Comment (provishalk) [reply src/pages/Registration/Registration.js]: donw

File: src/pages/Registration/Registration.js (hunk 8 above)
Comment (provishalk) [reply src/pages/Registration/Registration.js]: done

File: src/pages/Registration/Registration.js (hunk 7 above)
Comment (provishalk) [reply src/pages/Registration/Registration.js]: DONE

File: src/pages/Registration/Registration.js (hunk 9 above)
Comment (provishalk) [reply src/pages/Registration/Registration.js]: DONE

File: src/pages/Registration/Registration.js (hunk 10 above)
Comment (provishalk) [reply src/pages/Registration/Registration.js]: DONE

File: src/App.js [hunk 12]
```diff
@@ -1,24 +1,19 @@
...
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <>
+      <BrowserRouter>
+        {/* <Route path="/" exact component={} /> */}
```
Comment (shivdotpy) [inline src/App.js]: remove commented code

Comment (shivdotpy) [reply src/App.js]: Done

**Review by shivdotpy — APPROVED** [review]

**Pull request merged**